pymc/distributions/continuous.py

```diff
@@ -365,10 +365,13 @@ class Flat(Continuous):
 
     rv_op = flat
 
+    def __new__(cls, *args, **kwargs):
+        kwargs["initval"] = kwargs.get("initval", "moment")
```

Comment:
```suggestion
        kwargs.setdefault("initval", "moment")
```

Small nitpick, but this could be done with [setdefault](https://docs.python.org/3/library/stdtypes.html#dict.setdefault)